documentation/src/main/asciidoc/stories/assembly_server_rolling_upgrades.adoc

```diff
@@ -1,13 +1,12 @@
-[id='rolling_server_upgrades']
-:context: upgrade
-= Performing Rolling Upgrades for {brandname} Servers
+[id='rolling-upgrades']
+:context: rolling-upgrades
+= Performing rolling upgrades for {brandname} Server clusters
 Perform rolling upgrades of your {brandname} clusters to change between
-versions without downtime or data loss. Rolling upgrades migrate both your
-{brandname} servers and your data to the target version over Hot Rod.
+versions without downtime or data loss.
+Rolling upgrades migrate your {brandname} Server instances as well as your data to the target version over Hot Rod.
```

Comment:
What does "migrate your {brandname} Server instances" mean? Rolling Upgrades on bare metal only migrates the data, it does not deal with configuration, starting/stop/creating servers

Reply:
nice catch, thanks. I've reworded this.